src/app/GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -3154,7 +3154,20 @@ private void LaunchRebase(string command)
             });
 
             using FormProcess formProcess = new(UICommands, arguments: rebaseCmd, Module.WorkingDir, input: null, useDialogSettings: true);
-            formProcess.ProcessEnvVariables.Add("GIT_SEQUENCE_EDITOR", string.Format("sed -i -re '0,/pick/s//{0}/'", command));
+
+            const string envVarNameGitSequenceEditor = "GIT_SEQUENCE_EDITOR";
+            formProcess.ProcessEnvVariables.Add(envVarNameGitSequenceEditor, string.Format("sed -i -re '0,/pick/s//{0}/'", command));
+
+            const string envVarNameWslEnvControl = "WSLENV";
```

Comment:
should this be skipped for non WSL?
(there is no harm in this and that would add an if, but some may be confused. Just comment?)